warehouse/templates/pages/help.html

```diff
@@ -305,13 +306,13 @@ <h2>About</h2>
 
         <h3 id="maintainers">{{ maintainers() }}</h3>
         <p>
-          The new PyPI is powered by the Warehouse project; <a href="https://warehouse.readthedocs.io/">Warehouse</a> is an open source project developed under the umbrella of the Python Packaging Authority (PyPA) and supported by the Python Packaging Working Group (PackagingWG).
+          PyPI is powered by the Warehouse project; <a href="https://warehouse.readthedocs.io/">Warehouse</a> is an open source project developed under the umbrella of the Python Packaging Authority (PyPA) and supported by the Python Packaging Working Group (PackagingWG).
         </p>
         <p>
           The <a href="https://www.pypa.io/en/latest/">PyPA</a> is an independent group of developers whose goal is to improve and maintain many of the core projects related to Python packaging.
         </p>
         <p>
-          The <a href="https://wiki.python.org/psf/PackagingWG">PackagingWG</a> is a working group of the Python Software Foundation (PSF) whose goal is to raise and disburse funds to support the ongoing improvement of Python packaging. Most recently it <a href="https://pyfound.blogspot.com/2017/11/the-psf-awarded-moss-grant-pypi.html" target="_blank" rel="noopener">secured a grant from Mozilla</a> whose funding currently enables Warehouse maintainers to work on the new PyPI.
+          The <a href="https://wiki.python.org/psf/PackagingWG">PackagingWG</a> is a working group of the Python Software Foundation (PSF) whose goal is to raise and disburse funds to support the ongoing improvement of Python packaging. Most recently it <a href="https://pyfound.blogspot.com/2017/11/the-psf-awarded-moss-grant-pypi.html" target="_blank" rel="noopener">secured an award from Mozilla</a> whose funding enabled Warehouse maintainers to launch the new PyPI.
```

Comment:
"the new PyPI" is going to make less and less sense over time, perhaps something like "the current version of PyPI" or "the Warehouse project as PyPI"?